src/readbin/compare.ml

```diff
@@ -0,0 +1,133 @@
+open Core_kernel.Std
+open Cmdliner
+open Bap.Std
+open Bap_plugins.Std
+
+exception No_unstripped_file
+
+let print_source = function
+  | `bw -> print_endline "bw"
+  | _ -> print_endline "others"
+
+let source = [
+  "BW", `bw;
+  "SymTbl", `symtbl;
+  "User", `user;
+  "Ida", `ida;
+]
+
+let source_gt = [
+  "User", `user;
+  "SymTbl", `symtbl
+]
+
+let tool : _ Term.t =
+  let doc = "The source of the function start result that we are going to
+  evaluate. " ^ (Arg.doc_alts_enum source) in
+  Arg.(value & pos 1 (enum source) `bw & info [] ~docv:"tool" ~doc)
+
+let gt : _ Term.t =
+  let doc = "The ground truth source. One can direct to a user file or use the symbol
+  table from unstripped binary (-u required). " ^ (Arg.doc_alts_enum source_gt) in
+  Arg.(value & pos 2 (enum source_gt) `symtbl & info [] ~docv:"gt" ~doc)
+
+let symsfile : string option Term.t =
+  let doc = "The symbol table of binaries. This requires the binaries to use this file as symbols source." in
+  Arg.(value & opt (some non_dir_file) None & info ["syms"; "s"]
+         ~docv:"syms" ~doc)
+
+let unstrip_bin : string option Term.t =
+  let doc = "The unstripped binary." in
+  Arg.(value & opt (some non_dir_file) None & info ["unstrip"; "u"]
+         ~docv:"unstripped_bin" ~doc)
+
+let use_ida : string option Term.t =
+  let doc = "Use Ida to extract symbols from file. \
+             You can optionally provide path to IDA executable, \
+             or executable name." in
+  Arg.(value & opt (some string) None & info
```

Comment:
Or better, not to use this option at all. What is the reason to have `--use-ida` option, when you're going to compare ida with something. Suppose I'm a user, I read the man page and trying to figure out how to use this. I want to compare ida with ground truth. I have `Ida` keyword, and `--use-ida` option. What should I do? The copy-pasted doc says, that I should use `--use-ida` since it will extract symbols from file, but if I don't use it? Totally confusing. The better name would be `--ida-path` and the text should clearly states, that this is a hint for the program, on where to find the IDA binary.